aspnetcore/introduction-to-aspnet-core.md

```diff
@@ -4,246 +4,250 @@ author: tdykstra
 
-We recommend the following sequence of tutorials for an introduction to developing ASP.NET Core apps:
+ASP.NET Core 2.x is supported on .NET Framework versions that implement .NET Standard 2.0:
 
-1. Follow a tutorial for the app type you want to develop or maintain.
+* .NET Framework latest version is recommended.
+* .NET Framework 4.6.1 or later.
 
-   |App type  |Scenario  |Tutorial  |
-   |----------|----------|----------|
-   |Web app                   | Client-side web UI development |[Get started with Blazor](https://dotnet.microsoft.com/learn/aspnet/blazor-tutorial/intro) |
-   |Web app                   | New server-side web UI development |[Get started with Razor Pages](xref:tutorials/razor-pages/razor-pages-start) |
-   |Web app                   | Maintaining an MVC app |[Get started with MVC](xref:tutorials/first-mvc-app/start-mvc)|
-   |Web API                   | RESTful HTTP services |[Create a web API](xref:tutorials/first-web-api)&dagger; |
-   |Remote Procedure Call app | Contract-first services using Protocol Buffers |[Get started with a gRPC service](xref:tutorials/grpc/grpc-start) |
-   |Real-time app             | Bidirectional communication between servers and connected clients |[Get started with SignalR](xref:tutorials/signalr) |
+To help close the API gap from .NET Framework to .NET Core 2.x, the [Windows Compatibility Pack](/dotnet/core/porting/windows-compat-pack) made thousands of Windows-only APIs available. These APIs weren't available in .NET Core 1.x.
 
-1. Follow a tutorial that shows how to do basic data access.
+## Recommended learning path (TABLES LAYOUT OPTION)
 
-   |Scenario  |Tutorial  |
-   |----------|----------|
-   |New development        |[Blazor with Entity Framework Core](xref:blazor/tutorials/movie-database-app/index) |
-   |New development        |[Razor Pages with Entity Framework Core](xref:data/ef-rp/intro) |
-   |Maintaining an MVC app |[MVC with Entity Framework Core](xref:data/ef-mvc/intro) |
+<!-- AUTHOR NOTE: This is the original layout using tables. -->
 
-1. Read an overview of ASP.NET Core [fundamentals](xref:fundamentals/index) that apply to all app types.
+We recommend the following sequence of tutorials for an introduction to developing ASP.NET Core apps, or select the appropriate tutorial for a particular type of app that you want to build.
 
-1. Browse the table of contents for other topics of interest.
+Our tutorial for new developers and developers new to .NET is <xref:get-started>.
 
-&dagger;There's also an [interactive web API tutorial](/training/modules/build-web-api-net-core). No local installation of development tools is required. The code runs in an [Azure Cloud Shell](https://azure.microsoft.com/features/cloud-shell/) in your browser, and [curl](https://curl.haxx.se/) is used for testing.
+:::moniker range=">= aspnetcore-6.0"
 
-## Migrate from .NET Framework
+<!-- AUTHOR NOTE: I demoted RP/MVC to the bottom of the 
```

Comment:
We don't want to confuse new users with having to choose between Blazor, MVC, and Razor Pages, so that MVC & Razor Pages tutorials should be dropped from this list.